### PR TITLE
[MIG][10.0] website_hide

### PR DESCRIPTION
**Pull request by hbrunn**
migration of https://github.com/OCA/website/pull/332

### CODE REVIEW

File: website_hide/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017-2019 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Hide website",
+    "version": "10.0.1.0.0",
```
Comment (Nikul-Chaudhary) [inline website_hide/__manifest__.py:6]: Add `website` key

File: website_hide/data/ir_ui_menu.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
```
Comment (Nikul-Chaudhary) [inline website_hide/data/ir_ui_menu.xml]: Change <odoo>


no need <data>

**Review by Nikul-Chaudhary — COMMENTED** [review]
Improve Code

**Comment by simahawk** [comment]
tests on runbot:

BEFORE install

screenshot from 2019-02-01 08-38-42

AFTER install

screenshot from 2019-02-01 08-40-26

AFTER uninstalling cookie notice

image

**Comment by simahawk** [comment]
on travis there are still some website ui tests failing

**Comment by hbrunn** [comment]
@simahawk thanks for the screenshots, do you think I need to change something here?

As for the tests, that's already in 10: https://travis-ci.org/OCA/website/builds/458884205

**Review by simahawk — APPROVED** [review]
@hbrunn nothing to change. You could mention that would be better to uninstall some website modules like cookie notice, but I guess it's so evident that we don't need to say anything :)

**Review by NL66278 — APPROVED** [review]
:+1: LGTM and works in practice

**Pull request closed without merging**